bridge-fungible-token/src/events.sw

```diff
@@ -0,0 +1,26 @@
+library events;
+
+use std::{
+    contract_id::ContractId,
+    identity::Identity,
+    u256::U256,
+};
+
+pub struct RefundRegisteredEvent {
+    from: b256,
+    asset: b256,
+    amount: b256,
+}
+
+pub struct DepositEvent {
+    to: Address,
+    from: b256,
+    amount: u64,
+}
+
+pub struct WithdrawalEvent {
+    to: b256,
+    from: Identity,
+    amount: u64,
+    asset: ContractId,
```

Comment:
`asset` is not necessary since it will always just be the contract ID

Reply:
Ah yes, and the log receipt already includes the current contract id.